app/retail/emails.py

```diff
@@ -920,13 +920,13 @@ def render_new_bounty_roundup(to_email):
 Hey Gitcoiners,
 </p>
 <p>
-    We’re proud to announce that this week marks the launch of our newest product, Gitcoin Quests. Quests is an enjoyable, gamified way to learn about the Web3 ecosystem, compete with your friends, earn rewards, and level up your knowledge. We’ve been working hard for the past couple months and are excited to see how you like it. <a href="https://gitcoin.co/blog/gitcoin-quests-is-now-live/">To begin your adventure, click here.</a>
+    🌱 Gitcoin Grants CLR Round 3 was a massive, massive success. With wide ranging support from across the entire Ethereum ecosystem, we were blown away with the support you provided: 1,982 unique contributions to 68 projects worth $170,000. If you'd like to take a closer look at where the record $270k ended up, take a look at our blog post. <a href="https://gitcoin.co/blog/gitcoins-q3-match/">It's living here.</a>
 </p>
 <p>
-    ⏰ It’s almost that time. Web3 World is around the corner, and we’re stoked to offer another tremendous invitation for learning, earning, and of course, hacking. Watch out for a tweet later today with an official announcement post. Not following us on Twitter yet? <a href="https://twitter.com/gitcoin">Let’s fix that.</a>
+    🚨Kudos Alert!🚨 We've released a brand new feature with Kudos – the ability to mint your own. Like your favorite cappuccino machine, it lets you make your own delicious <strike>coffee</strike> Kudos all by yourself. If you have Kudos you want to put on the marketplace, all you've have to do is fill out a form, add your artwork, and watch as it makes its way to the Gitcoin Kudos marketplace! How neat is that. Check out the self serve feature <a href="https://gitcoin.co/kudos/new">here.</a>
```

Comment:
🚨New Kudos Feature Alert!🚨 We've released a brand new feature for Kudos – the ability to mint your own *without asking us to help*. Like your favorite cappuccino machine, it lets you make your own delicious <strike>coffee</strike> Kudos all by yourself. If you have Kudos you want to put on the marketplace, all you've have to do is fill out a form, add your artwork, and watch as it makes its way to the Gitcoin Kudos marketplace! How neat is that. Check out the self serve feature